### ISSUE TITLE
Failed start micro server

### ISSUE THREAD

**Issue by rozaq14996** [micro/micro#980]
Win 10 Pro 1909
Install micro using (powershell -Command "iwr -useb https://raw.githubusercontent.com/micro/micro/master/scripts/install.ps1 | iex")

i got some error when start micro server, and cannot register new micro service
but when i try to start "micro web" individualy works properly

{"id":"go.micro.client","code":500,"detail":"service go.micro.auth: not found","status":"Internal Server Error"}
2020-06-30 19:04:51  file=runtime/service.go:217 level=error service=micro Service micro.web terminated with error exit status 1
{"id":"go.micro.client","code":500,"detail":"service go.micro.auth: not found","status":"Internal Server Error"}
2020-06-30 19:04:51  file=runtime/service.go:217 level=error service=micro Service micro.api terminated with error exit status 1
{"id":"go.micro.client","code":500,"detail":"service go.micro.auth: not found","status":"Internal Server Error"}
2020-06-30 19:04:51  file=runtime/service.go:217 level=error service=micro Service micro.proxy terminated with error exit status 1

**Comment by rozaq14996** [comment]
And, if i running in win 7 using virtual box, there is no problem

**Comment by egruble** [comment]
Same issue for me on Windows Pro build 19041 (2004). I noticed `micro list services` didn't show any services registering, so on a hunch I tracked this down: [How to enable mDNS on Windows 10](https://superuser.com/questions/1330027/how-to-enable-mdns-on-windows-10-build-17134). The one-liner `REG ADD "HKLM\Software\Policies\Microsoft\Windows NT\DNSClient" /V "EnableMulticast" /D "0" /T REG_DWORD /F` from an admin command prompt and a reboot did the trick for me.
